### PR TITLE
Preparing for 0.11.0 Re-release.

### PR DESCRIPTION
**Pull request by Narsil**

### CODE REVIEW

File: bindings/python/CHANGELOG.md [hunk 1]
```diff
@@ -6,10 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [0.11.0]
 
+### Fixed
+
+- [#585] Conda version should now work on old CentOS
```
Comment (n1t0) [inline bindings/python/CHANGELOG.md:11]: You also need to add links at the end of this file

**Review by n1t0 — APPROVED** [review]
Awesome! Thank you for fixing the conda release! I just commented on a couple details

File: .github/workflows/python-release-extra.yml [hunk 2]
```diff
@@ -20,6 +20,9 @@ jobs:
     steps:
       - uses: actions/checkout@v1
 
+      - name: Upgrade libssl
+        run: sudo apt-get install -y libssl-dev
```
[the remark this answers is not in the document]
Comment (Narsil) [reply .github/workflows/python-release-extra.yml:24]: Yes, otherwise  `docker pull` is unhappy.

File: bindings/python/CHANGELOG.md (hunk 1 above)
Comment (Narsil) [reply bindings/python/CHANGELOG.md:11]: Spoiled by GH usual autolinks :)

File: bindings/python/CHANGELOG.md [hunk 3]
```diff
@@ -6,10 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- [#844] Fixing interaction between `is_pretokenized` and `trim_offsets`.
+- [#846] Clippy fixes
```
Comment (n1t0) [inline bindings/python/CHANGELOG.md:13]: No need to include this

Comment (Narsil) [reply bindings/python/CHANGELOG.md]: Removed

**Pull request merged**